jetbrains-ultimate/tst/software/aws/toolkits/jetbrains/services/redshift/RedshiftUtilsTest.kt

```diff
@@ -13,12 +14,13 @@ import software.aws.toolkits.core.utils.RuleUtils
 import software.aws.toolkits.jetbrains.core.MockResourceCacheExtension
 import software.aws.toolkits.jetbrains.core.region.getDefaultRegion
 import software.aws.toolkits.jetbrains.services.sts.StsResources
-import software.aws.toolkits.jetbrains.testFramework.ProjectExtension
 
 class RedshiftUtilsTest {
-    @JvmField
-    @RegisterExtension
-    val projectRule = ProjectExtension()
+    companion object {
```

Comment:
Why is this moved to a companion object?

Reply:
it's the equivalent of a class rule. the extension only runs before/after the entire suite